presto-spi/src/test/java/com/facebook/presto/spi/type/TestDecimals.java

```diff
@@ -127,6 +132,51 @@ public void testRejectNoDigits()
         assertParseFailure("-.");
     }
 
+    @Test
+    public void testEncodeShortScaledValue()
+    {
+        assertEquals(encodeShortScaledValue(new BigDecimal("2.00"), 2), 200L);
+        assertEquals(encodeShortScaledValue(new BigDecimal("2.13"), 2), 213L);
+        assertEquals(encodeShortScaledValue(new BigDecimal("172.60"), 2), 17260L);
+        assertEquals(encodeShortScaledValue(new BigDecimal("2"), 2), 200L);
+        assertEquals(encodeShortScaledValue(new BigDecimal("172.6"), 2), 17260L);
+
+        assertEquals(encodeShortScaledValue(new BigDecimal("-2.00"), 2), -200L);
+        assertEquals(encodeShortScaledValue(new BigDecimal("-2.13"), 2), -213L);
+        assertEquals(encodeShortScaledValue(new BigDecimal("-2"), 2), -200L);
+    }
+
+    @Test
+    public void testEncodeScaledValue()
+    {
+        assertEquals(encodeScaledValue(new BigDecimal("2.00"), 2), sliceFromBytes(200, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0));
```

Comment:
nit:
maybe sth like
```
sliceFrom("a000000000000003232032030230")

Reply:
If i `s/200/0xA0`, i will get almost the same text (except I have the commas, which are not that bad (slightly helpful) and can define a "constant" `minus`, used below

Reply:
ok